server/src/controllers/ScheduleController.js

```diff
@@ -0,0 +1,102 @@
+const ScheduleServices = require('../services/ScheduleServices');
+
+const ScheduleController = {
+  /**
+   * Get an array of events for a frosh group.
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Function} next
+   * @return {Object[]}
```

Comment:
```suggestion

```
This function does not return anything, so having a return value in the doc comment is misleading